cmd/get_logs.go

```diff
@@ -347,6 +380,28 @@ func (glc *getLogsCmd) downloadLogs(node v1.Node, client *ssh.Client) (string, e
 	return "", nil
 }
 
+func (glc *getLogsCmd) validateLogScript() error {
+	if glc.linuxScriptPath == "" && !glc.cs.Properties.MasterProfile.IsVHDDistro() {
+		if glc.controlPlaneOnly {
+			return errors.Errorf("No log collection script found for control plane nodes")
+		} else {
+			log.Warnf("Skipping master nodes nodes as no log collection script found")
```

Comment:
- `Warn` instead of `Warnf`
- `master nodes nodes`
- let's pick between `master` OR `control plane` and use it consistently

Reply:
Updated